src/components/Main/index.js

```diff
@@ -89,34 +88,22 @@ const Main = () => (
           </Button>
         </a>
       </div>
-      {/*
-      // TokenSale Box
-      <div className="token-sale-box" style={{overflow: 'hidden', marginTop: '40px'}}>
-        <div className="token-sale-header" style={{display: 'flex', paddingLeft: '25px', alignItems: 'center', verticalAlign: 'middle'}}>
-          <div className="circle" style={{marginRight: '25px'}} />
-          <div style={{verticalAlign: 'center', height: '36px', padingTop: '5px'}}>Token Sale</div>
+      <div className="token-sale-box" style={{position: 'relative', overflow: 'hidden', marginTop: '40px'}}>
+        <div className="token-sale-header" style={{display: 'flex', paddingLeft: '199px', alignItems: 'center', verticalAlign: 'middle'}}>
+          <div style={{verticalAlign: 'center', height: '36px', padingTop: '5px'}}>Token Distribution</div>
         </div>
-        <div className="token-sale-box-body" style={{display: 'flex', alignItems: 'center', flexDirection: 'column', width: '100%', height: '100%', overflow: 'hidden'}}>
-          <div className="graph" style={{marginTop: '45px'}}>
-            <div className="graph-left" style={{width: `${tokenSales[4].text.split('%')[0]}%`}}>
-            </div>
-            <div className="">
-            </div>
+        <img className="ares-coin" src="/images/arestech-coin.png" alt="arestech-coin" style={{position: 'absolute', width:"198", height:"198", left: '5px', top: '20px'}}/>
+        <div className="token-sale-box-body" style={{display: 'flex', paddingLeft: '199px', flexDirection: 'column', width: '100%', height: '100%', overflow: 'hidden'}}>
+          <div className="token-sale-header2" style={{ marginTop: '15px', height: '38px', fontSize: '40px', fontWeight: 'bold', color: '#001ada'}}>
+            Begins 09.2018
```

Comment:
The multi-lang support is already in master, if you rebase on latest master you should be able to use it. These texts guess need to be be extracted in the `src/data/messages/en.js`. Same with the one below.